common/mariadb/templates/deployment.yaml

```diff
@@ -22,9 +19,7 @@ spec:
   template:
     metadata:
       labels:
-        app: {{ include "fullName" . }}
-        name: {{ template "fullName" . }}
```

Comment:
Removal of these two metrics from the deployment, unfortunately, will break the following dashboards:
- "MariaDB InnoDB Metrics"
- "MariaDB Overview"

Both of them depend either on `name` or `app` label.
These dashboards use `label_values()` function, which doesn't support queries, which could be used to support any of new or old labels.

Reply:
Only the `MariaDB Overview` dashboard is affected. We can create a new version of the dashboard in case we do not find a better way to support both label types. After all services have been migrated we can removed the old dashboard...